index.bs

```diff
@@ -10196,29 +10196,25 @@ thread and the rendering thread.
 		  [$StructuredDeserialize$](<code>optionsSerialization</code>, the
 		  current Realm).
 
-	2. Let <var>processorConstructor</var> be the result of looking up
+	1. Let <var>processorConstructor</var> be the result of looking up
 		<var>nodeName</var> on the {{AudioWorkletGlobalScope}}'s <a>node
 		name to processor definition map</a>.
 
-	3. If <var>processorConstructor</var> is <code>undefined</code>,
+	1. If <var>processorConstructor</var> is <code>undefined</code>,
 		throw a {{NotSupportedError}} DOMException.
 
-	4. Let <var>processor</var> be the result of
+	1. Let <var>processor</var> be the result of
 		Construct(<var>processorConstructor</var>, « <var>options</var> »).
 
-	5. If <var>processor</var> does not implement the
-		{{AudioWorkletProcessor}} interface, throw an
-		{{InvalidStateError}} DOMException.
+	1. Set <var>processor</var>'s {{AudioWorkletProcessor/port}} to <var>processorPort</var>.
 
-	6. Set <var>processor</var>'s {{AudioWorkletProcessor/port}} to <var>processorPort</var>.
-
-	7. Set <var>processor</var>'s <a>node reference</a> to
+	1. Set <var>processor</var>'s <a>node reference</a> to
 		<var>node</var>.
 
-	8. Set <var>node</var>'s <a>processor reference</a> to
+	1. Set <var>node</var>'s <a>processor reference</a> to
 		<var>processor</var>.
 
-	9. <a>Queue a task</a> the <a>control thread</a> to set the associated
+	1. <a>Queue a task</a> the <a>control thread</a> to set the associated
```

Comment:
Since you're modifying this anyway: "Queue a task the" -> "Queue a task to the".

Reply:
Done.